packages/contracts/scripts/shared/constants/mnemonic.ts

```diff
@@ -0,0 +1 @@
+export const TEST_MNEMONIC = "test test test test test test test test test test test junk"
```

Comment:
Seems odd to put this in its own file. I thought these were just hardcoded in the hardhat.config.ts files (and presumably the equivalent for Forge)

Reply:
Oh, you're right. I will replace it with the public one which ganache uses. 

https://github.com/ubiquity/ubiquity-dollar/pull/261/commits/3af10c5893e2299a219b6f34274bfd069d5da371

Reply:
Why not just use the same test one? It's already in the README.

Reply:
> Why not just use the same test one? It's already in the README.

oh i think i misunderstood you. so you were saying that we don't put this test mnemonic in its own file because these were already hardcoded in the hardhat.config.ts and you also assumed that would be same in forge ---- if this is right, Forge doesn't have such mnemonics hardcoded. that is why i put it in the file.

Reply:
I understand now. Perhaps it could be consolidated into a larger shared constants file? Your decision there. 

Also I want to clarify that you apparently changed the mnemonic to a new one:
```
"horn hammer original lemon chapter weird gun pond fortune blush cupboard cat" // MUST use this mnemonic only on testnets
```

Why did you use this new one instead of just leaving the old one?
```
"test test test test test test test test test test test junk"
```

Reply:
> `test test test test test test test test test test test junk`

Isn't this already in the README??? I guess its there and you were asking to use the same one in README. 
@pavlovcik